### PR TITLE
Auto add layout=responsive if missing

### PR DESCRIPTION
**Pull request by sebastianbenz**
Fixes https://github.com/ampproject/amp-toolbox/issues/959

### CODE REVIEW

**Review by patrickkettner — APPROVED** [review]

File: packages/optimizer/lib/transformers/ApplyLayout.js [hunk 1]
```diff
@@ -181,6 +181,12 @@ module.exports = {
       log.debug('cannot perform SSR: unsupported layout', layout);
       return false;
     }
+    // Transformed AMP validation requires layout attribute to be set
+    // See https://github.com/ampproject/amp-toolbox/issues/959
+    if (layout && layout === 'responsive') {
+      console.log('XXX setting custom layout ', layout);
```
Comment (patrickkettner) [inline packages/optimizer/lib/transformers/ApplyLayout.js:187]: why `console.log` vs the `log` util?

**Comment by patrickkettner** [comment]
LGTM 

wondering why responsive is the assumed default, and if there is a chance that having a different configurable default would make sense?

File: packages/optimizer/lib/transformers/ApplyLayout.js (hunk 1 above)
Comment (sebastianbenz) [reply packages/optimizer/lib/transformers/ApplyLayout.js:187]: removed

**Comment by sebastianbenz** [comment]
`responsive` is not the default. It's only that the validator does not auto infer the responsive layout as the server-side-rendering does.

**Pull request merged**